src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Report.tsx

```diff
@@ -155,6 +162,8 @@ export const Report = ({ activity, identifiers = [], versions = [] }: ReportProp
 
         setSubscalesFrequency(subscalesFrequency);
         setResponseOptions(formattedResponses);
+      } catch (error) {
+        console.warn(error);
```

Comment:
Can we put an error message on the user interface to show that something went wrong in the BE side?